website/src/pages/index.js

```diff
@@ -197,10 +198,10 @@ function Home() {
             {QUOTES.map(quote => (
               <div className="col" key={quote.name}>
                 <div className="avatar avatar--vertical margin-bottom--sm">
-                  <img
+                  <Image
                     alt={quote.name}
                     className="avatar__photo avatar__photo--xl"
```

Comment:
The resulting image does not have the border radius because `Image` component is actually a <div>. I might have to fix this on Infima end and add `overflow: hidden` to `avatar__photo`

Reply:
was going to point this out. my bad i gotta sleep now